sample_integration/sample_integration.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,363 @@
+			buildActionMask = 2147483647;
+			files = (
+				72540A082489A16400DCB938 /* Embrace-Info.plist in Resources */,
+				72540A0024899BA800DCB938 /* LaunchScreen.storyboard in Resources */,
+				72540A0C248D78B600DCB938 /* README.md in Resources */,
+				725409FD24899BA800DCB938 /* Assets.xcassets in Resources */,
+				725409FB24899BA700DCB938 /* Main.storyboard in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXSourcesBuildPhase section */
+		725409EA24899BA700DCB938 /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				725409F224899BA700DCB938 /* AppDelegate.swift in Sources */,
+				725409F424899BA700DCB938 /* SceneDelegate.swift in Sources */,
+				725409F624899BA700DCB938 /* MasterViewController.swift in Sources */,
+				725409F824899BA700DCB938 /* DetailViewController.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXVariantGroup section */
+		725409F924899BA700DCB938 /* Main.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				725409FA24899BA700DCB938 /* Base */,
+			);
+			name = Main.storyboard;
+			sourceTree = "<group>";
+		};
+		725409FE24899BA800DCB938 /* LaunchScreen.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				725409FF24899BA800DCB938 /* Base */,
+			);
+			name = LaunchScreen.storyboard;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		72540A0224899BA800DCB938 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = dwarf;
```

Comment:
should we enable dsym generation by default for crash symbolication?

Reply:
oh yeah I didn't add any of that, coming right up.

Reply:
i only brought it up because it is one of the potential stumbling blocks for seeing symbolication work

Reply:
It's important for this.  One twist is how to deal with api_key and app_token.  I put placeholders, this causes us to not work out of the box so I put a big comment in a few different places about replacing that with your own.  

Might be interesting to explore a kind of auto-provisioning setup.  Where this project runs, doesn't have a key so it auto-provisions itself against the server for some kind of time-limited trial or something.  Then it displays the url for the user to go view their data -- when they visit the url we force them to complete registration.  It's a bit of work but greatly reduces the "try-it-out" friction.

Reply:
yeah, that's a cool idea